spec/forecasts/forecast.js

```diff
@@ -1,9 +1,79 @@
+import {Forecast} from "../../src/forecasts/forecast.js";
+import * as weather from "weather-js";
+import {staticWeather} from "../static-resource/weather.json"; //Mock data
 
-describe("here1>", () => {
+describe("Forecast", () => {
+  let forecast;
 
-  it('will assert true to equal true', () => {
+  describe("getWeather", () => {
+    let expectedErrMsg;
```

Comment:
call full name expectedErrorMessage?

Reply:
Quite a long variable name anyhow was trying to make it shorter. Can change it if you feel really needs to be full name personally think it is descriptive enough though?

Reply:
Good Practice to use full names, don't worry how long the variable name as the more explicit it is the better.